addons/godot-xr-tools/assets/PlayerBody.gd

```diff
@@ -483,7 +483,7 @@ func _get_configuration_warning():
 	return ""
 
 ## Find the Player Body from a player node and an optional path
-static func get_player_body(node: Node, var path: NodePath = "") -> PlayerBody:
+static func get_player_body(node: Node, var path := NodePath("")) -> PlayerBody:
```

Comment:
Nitpicking here but should we do:
```
static func get_player_body(node: Node, var path : NodePath = NodePath("")) -> PlayerBody:
```
for consistency? it's a little more verbose but it seems that Godot 4 really does prefer typed variables over untyped even if in this case it can easily infer it.

Reply:
Agreed, I switched them all over to the long-form style.